app/controllers/candidate_interface/gcse_details_controller.rb

```diff
@@ -0,0 +1,15 @@
+module CandidateInterface
+  class GcseDetailsController < CandidateInterfaceController
+    def edit
+      subject = params[:subject]
+
+      @application_qualification = ApplicationQualification.create(subject: subject, application_form: current_candidate.current_application, level: 'gcse')
```

Comment:
Should we add a `enum subject` to the model? E.g.:

```ruby
enum subject: {
  maths: 'maths',
  english: 'english',
  science: 'science',
}
```

This would make it more resilient to spoofing the `params[:subject]`.

Reply:
I agree! is it ok to add it in the following PR?

Reply:
Done